scripts/tests/java/commissioning_test.py

```diff
@@ -41,18 +41,21 @@ def __init__(self, thread_list: typing.List[threading.Thread], queue: queue.Queu
 
         parser.add_argument('command', help="Command name")
         parser.add_argument('-t', '--timeout', help="The program will return with timeout after specified seconds", default='200')
-        parser.add_argument('-a', '--address', help="Address of the device")
+        parser.add_argument('-a', '--address', help="Address of the remote device")
```

Comment:
Why do we want this supported?

CHIP spec generally does not allow direct IP address specification or port ... address and port are discovered via MDNS.

Reply:
How do we test pairing if we need to start commissioner and all-cluster-apps within their own container or VMs?   
I try to align java chip-tool with c++ chip-tool which currently use the following cmd to pair a controller to a remote device.
`./chip-tool pairing ethernet 12344321 20202021 3840 192.168.86.42 5540`

Reply:
I would expect the container VM to be attached to the same network and mdns to work, so discriminator/mdns search should work.

Reply:
Guides say to use 'onnetwork':

https://github.com/project-chip/connectedhomeip/blob/master/examples/chip-tool/README.md#pair-a-device-over-ip

I don't think we should have ethernet at all ... we used to have that before mdns was working.

Reply:
Pinged in slack to be sure ... however I believe that we should not have (maybe even delete completely) the `ethernet` option ... we should not have by address/port options.